pkg/logs/input/listener/udp_nix_test.go

```diff
@@ -75,3 +75,49 @@ func TestUDPShoulDropTooBigMessages(t *testing.T) {
 
 	listener.Stop()
 }
+
+func TestUDPShoulProperlyCollectLogSplitPerDatadgram(t *testing.T) {
```

Comment:
Should we also test the edge case where the message length is == the frame size ?